packages/react-component-library/src/styled-components/partials/StyledOuterWrapper.tsx

```diff
@@ -43,15 +39,15 @@ export const StyledOuterWrapper = styled.div<StyledOuterWrapperProps>`
         ${defaults};
         border: ${BORDER_WIDTH[$size]} solid ${color('action', '500')};
         box-shadow: 0 0 0 ${BORDER_WIDTH[$size]} ${color('action', '100')};
-        margin: ${MARGIN[$size]};
```

Comment:
The negative margin was annoying me too :smile: But this will move the border to the inside on the text input etc., is that what we want?

Think we can keep it outside using multiple shadows (this is for the focus state):

```
        // box-sizing: border-box;

        ...

        // border: ${BORDER_WIDTH[$size]} solid ${color('action', '500')};
        box-shadow: 0 0 0 ${BORDER_WIDTH[$size]} ${color('action', '500')},
          0 0 0 ${math(`${BORDER_WIDTH[$size]} * 2`)} ${color('action', '100')};
```

(using `import { math } from 'polished'` there but `calc()` would work too).